dbms/src/Flash/Planner/PhysicalPlanNode.h

```diff
@@ -56,6 +60,10 @@ class PhysicalPlanNode
 
     virtual void transform(DAGPipeline & pipeline, Context & context, size_t max_streams);
 
+    virtual void transform(OperatorPipelineGroupBuilder & /*group_builder*/, Context & /*context*/, size_t /*concurrency*/);
```

Comment:
As I mentioned in my previous comment, this `transform` name conflicts with the other usage in `Operator` and is misleading. Maybe we can rename it to `buildPipelineExec`?

Reply:
ok, updated.